contrib/pebble_env/pebble.h

```diff
@@ -12,6 +12,11 @@
 #include <string.h>
 #include <inttypes.h>
 
+#define NGFX_PREFERRED_free free
+#define NGFX_PREFERRED_calloc calloc
+#define NGFX_PREFERRED_malloc malloc
+#define NGFX_PREFERRED_resource_load resource_load
```

Comment:
Not sure if I missed it, but I don't see this being used anywhere, but is required in a macro later?

Reply:
The macros in this file are used only for the PC testrunner, RebbleOS has to provide its own ones.
The change is meant to move the selection of memory functions out of ngfx.